tests/common/file_utils.rs

```diff
@@ -58,3 +63,10 @@ pub fn create_file_with_content(path: &PathBuf, content: &str) -> () {
     file.write_all(content.as_bytes())
         .expect(&format!("cannot write to file {:?}", path));
 }
+
+pub fn read_file(path: &PathBuf) -> String {
+    let mut file = File::create(&path).unwrap();
+    let mut buffer = String::new();
+    std::fs::read_to_string(&mut buffer);
+    buffer
```

Comment:
I don't understand what this function is doing. `file` is not used at all. What are you reading?

The function [`std::fs::read_to_string`](https://doc.rust-lang.org/stable/std/fs/fn.read_to_string.html) does not read in the buffer here... please read the doc.

Reply:
Yes this is mistake, removing method